research/static/research/js/research.js

```diff
@@ -11,6 +11,8 @@ $(document).ajaxSend(function (e, request, settings) {
 })
 
 let preservableFormatsLists = null
+let folderCreateTooltip
+let uploadMenuTooltip
```

Comment:
Is there a better way to do this without using globals?

Reply:
When we start using ES6 or later, we can probably clean this up by splitting research.js into more appropriate modules. To reduce the number of globals and organize the code and reduce the risk of naming conflicts, we could use a single global dict instead of using a bunch of global variables.